packages/now-cli/test/dev/integration.js

```diff
@@ -1568,6 +1568,14 @@ test(
   })
 );
 
+test(
+  '[vercel dev] 29-vercel-region should assign VERCEL_REGION env var',
+  testFixtureStdio('29-vercel-region', async testPath => {
+    await testPath(200, `/api/now-region`, /NOW_REGION is (dev1|sfo1)/m);
+    await testPath(200, `/api/vercel-region`, /VERCEL_REGION is (dev1|sfo1)/m);
```

Comment:
This won't really work in production because `VERCEL_REGION` is only set if configured as a System Env Var for the Project.